scene/resources/material.cpp

```diff
@@ -507,6 +507,8 @@ SelfList<BaseMaterial3D>::List BaseMaterial3D::dirty_materials;
 HashMap<BaseMaterial3D::MaterialKey, BaseMaterial3D::ShaderData, BaseMaterial3D::MaterialKey> BaseMaterial3D::shader_map;
 BaseMaterial3D::ShaderNames *BaseMaterial3D::shader_names = nullptr;
 
+constexpr const char *SHADER_UV_PARAMS[] = { "UV", "UV2", "CUSTOM0.xy", "CUSTOM0.zw", "CUSTOM1.xy", "CUSTOM1.zw", "CUSTOM2.xy", "CUSTOM2.zw" };
```

Comment:
```suggestion
static constexpr const char *SHADER_UV_PARAMS[] = { "UV", "UV2", "CUSTOM0.xy", "CUSTOM0.zw", "CUSTOM1.xy", "CUSTOM1.zw", "CUSTOM2.xy", "CUSTOM2.zw" };
```
To keep local and prevent namespace pollution

Reply:
I could alternatively put this in an anonymous namespace if you'd prefer?

Reply:
I'd say static, it's what we normally do and it's cleaner